### PR TITLE
added Word Shark game

### PR DESCRIPTION
**Pull request by MattDClarke**

### CODE REVIEW

File: src/helpers/gamesInfo.js [hunk 1]
```diff
@@ -17,6 +17,10 @@ import SleepingBears from "@Game/SleepingBears";
 import MissingLetter from "@Game/MissingLetter";
 import FixTheMistake from "@Game/FixTheMistake";
 import WhatsMissing from "@Game/WhatsMissing";
+import WordShark from "@Game/WordShark/WordShark";
```
Comment (dastrong) [inline src/helpers/gamesInfo.js:20]: You can remove that additional `/WordShark`

File: src/helpers/gamesInfo.js [hunk 2]
```diff
@@ -1446,4 +1450,47 @@ export default [
...
+  {
+    router: {
+      path: "/game/wordshark",
+      component: WordShark,
+      icon: "odnoklassniki",
+    },
+    info: {
+      title: "Word Shark",
+      skills: ["Reading", "Speaking"],
+      dataUsed: ["Vocabulary"],
```
Comment (dastrong) [inline src/helpers/gamesInfo.js:1462]: Would it be possible to also include expressions? Think that'd be nifty to have when someone reviews multiple lessons.

Let me know in a comment if that's something you'd want to try including.

File: src/helpers/gamesInfo.js [hunk 3]
```diff
@@ -1446,4 +1450,47 @@ export default [
...
+      description: "Word Guessing Game",
+      // image: `${baseURL}/v1570941133/TeacherSite/Games/Word_Shark.svg`,
+      image: WordSharkImg,
+      attachments: false,
+      audio: false,
+      completed: true,
+    },
+    keyCuts: [
```
Comment (dastrong) [inline src/helpers/gamesInfo.js:1470]: I was hoping users could click the letter that was said on their keyboard also. Just need to use the key events hook and dispatch a `Handle_Guess` if it's a letter and it hasn't been guessed already.

Let me know in a comment if that's something you'd want to try including.

**Review by dastrong — CHANGES_REQUESTED** [review]
You did a great job mate. I haven't looked at this codebase in awhile, but just looking at the code it looks like you handled it well. And now you've got some experience working on an 'old' unfamiliar codebase!

I left some comments below for you to check out.

File: src/helpers/gamesInfo.js (hunk 2 above)
Comment (MattDClarke) [reply src/helpers/gamesInfo.js:1462]: That would be possible, I would need to change the layout. The problem is that the letter buttons and image take up a lot of space so a full sentence might be quite small on the screen. Have you got a suggestion on how to alter the layout?

File: src/helpers/gamesInfo.js (hunk 3 above)
Comment (MattDClarke) [reply src/helpers/gamesInfo.js:1470]: I'll add it

**Comment by dastrong** [comment]
I'm gonna merge this into my `dev` branch. 

You can still push those changes mentioned above to this PR, when they're ready.

**Pull request merged**